app/Http/Transformers/EventTransformer.php

```diff
@@ -15,18 +15,22 @@ class EventTransformer extends TransformerAbstract
      */
     public function transform(Event $event)
     {
+        $content = [];
+        $contentString = substr($event->content, 1, -1);
```

Comment:
This field should be saved as JSON in the database, so we should be able to use `json_decode($event->content)` here to prepare it for the response. In the future, we should look into using either Eloquent's built-in `array` casting or an accessor & mutator to handle this within the `Event` model.

Reply:
updated this!